admin/client.go

```diff
@@ -201,13 +201,7 @@ func (c *Client) ListChangeSummaries(
 		return nil, err
 	}
 
-	lastSeq := changes[len(changes)-1].ID().ServerSeq()
-	from, _ := types.GetChangesRange(types.Paging[uint64]{
-		Offset:    previousSeq,
-		PageSize:  int(pageSize),
-		IsForward: isForward,
-	}, lastSeq)
-	seq := from - 1
+	seq := changes[0].ServerSeq()
```

Comment:
Is there any problem with changing this code?
CC) @chacha912

Reply:
To explain why I've implemented this, when there's a `--backend-snapshot-with-purging-changes` flag, the changes of the front are deleted, so I want to create a history based on the changes that the ListChanges succeeded in loading from db .
I thought there would be no problem because the change is imported from the db based on GetChangesRange in ListChanges.

I'd appreciate it if you could point out something that I missed.

Reply:
I like it. 
It didn't occur to me that I can get sequence by `changes[0].ServerSeq()`. 
Thanks :)